src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -546,7 +550,7 @@ class EthService(
       reportActive()
       val bestBlock = blockchain.getBestBlock()
       val response: ServiceResponse[GetWorkResponse] =
-        getOmmersFromPool(bestBlock.hash).zip(getTransactionsFromPool).map { case (ommers, pendingTxs) =>
+        Task.parZip2(getOmmersFromPool(bestBlock.hash), getTransactionsFromPool()).map { case (ommers, pendingTxs) =>
```

Comment:
IMHO I like to use "for comprehension" more than the "zip" function. What do you think?

Reply:
parZip2 is running actions in parallel, for comprehension enforce sequential behavior. So probably parZip is better.

Reply:
excelent!